pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -543,6 +698,7 @@ func (r *ReconcileComplianceScan) phaseAggregatingHandler(h scanTypeHandler, log
 	}
 
 	instance.Status.Phase = compv1alpha1.PhaseDone
+	instance.Status.EndTimestamp = &metav1.Time{Time: time.Now()}
```

Comment:
What's the difference between doing this in the aggregation phase versus the done phase?

Reply:
the scan is stilling running in the aggregation phase?